iteration.md

```diff
@@ -1,14 +1,14 @@
-# Iteration
+# Iteração
 
-**[You can find all the code for this chapter here](https://github.com/quii/learn-go-with-tests/tree/master/for)**
+**[Você pode encontrar todo o código desse capitulo aqui](https://github.com/quii/learn-go-with-tests/tree/master/for)**
```

Comment:
Faltando acento em capítulo :dancer: também já dá para atualizar o link para o capítulo para [esse aqui](https://github.com/larien/learn-go-with-tests/tree/master/for).